vitrina/structure/forms.py

```diff
@@ -305,6 +354,18 @@ class ModelCreateForm(forms.ModelForm):
         widget=forms.RadioSelect,
         choices=MODEL_LEVEL_CHOICES,
     )
+    visibility = ModelChoiceTypeField(
+        label=_("Metaduomenų matomumas"),
+        required=False,
+        queryset=Visibility.objects.all(),
+        widget=forms.RadioSelect,
+    )
+    status = ModelChoiceTypeField(
+        label=_("Statusas"),
```

Comment:
What would you think about this? Unless there is some hard requirement, that it should be the less-lithuanianly "Statusas":

```suggestion
        label=_("Būsena"),
```